src/main/java/at/ac/tuwien/kr/alpha/config/SystemConfig.java

```diff
@@ -60,6 +61,8 @@
 	public static final String DEFAULT_GROUNDER_TOLERANCE_CONSTRAINTS = GrounderHeuristicsConfiguration.STRICT_STRING;
 	public static final String DEFAULT_GROUNDER_TOLERANCE_RULES = GrounderHeuristicsConfiguration.STRICT_STRING;
 	public static final boolean DEFAULT_GROUNDER_ACCUMULATOR_ENABLED = false;
+	public static final boolean DEFAULT_ENABLE_RESTARTS = false;
+	public static final String DEFAULT_PHASE_INITIALIZER = PhaseInitializerFactory.InitialPhase.RULESTRUEATOMSFALSE.name().toLowerCase();
```

Comment:
This actually causes several other problems, because the `InitialPhase` needs access to the `Random` and `AtomStore` of the system, both are not known/created at this point. Do you have any idea for circumventing this problem?

Reply:
I don´t see the problem. `InitialPhase` is just an enum and doesn´t need to access anything. Please accept #253.

Reply:
Dang ..., yes, of course we don't need the instance of the phase initializer but just the enum.